Program.cs

```diff
@@ -1,21 +1,52 @@
 ﻿using MiniGame.App;
-using MiniGame.App.Warriors;
 using System.Text;
+using MiniGame.App.HW2;
 
 Console.OutputEncoding = Encoding.Unicode;
 
-var warrior1 = Menu.Hero(1);
-var warrior2 = Menu.Hero(2);
+do
+{
+    List<Warrior> warriors = new List<Warrior>();
 
-int addHero1 = SuperPowerGenerator.GetRandom();
-Console.WriteLine("\n Що ви хочете покращити першому Герою №1 на [" + addHero1 + "] пунктів");
-Menu.Update(warrior1, addHero1);
+    int iter = 1;
+    int players = NumberOfPlayers.Players();
 
-int addHero2 = SuperPowerGenerator.GetRandom();
-Console.WriteLine("\n Що ви хочете покращити першому Герою №2 на [" + addHero2 + "] пунктів");
-Menu.Update(warrior2, addHero2);
+    while (warriors.Count() < players)
+    {
+        warriors.Add(Menu.Hero(warriors.Count() + 1));
+        
+        int addToHero = SuperPowerGenerator.GetRandom();
+        Console.WriteLine($"\n Що ви хочете покращити Герою №{iter--} на [{addToHero}] пунктів");
+        Menu.Update(warriors.Skip(iter++).First(), addToHero);
```

Comment:
It is not a best solution to get hero. You can assign hero to local field (line 16) and use that field